content/2020/09-29-changes-to-oauth-2-app-redirect-url-requirements.md

```diff
@@ -0,0 +1,47 @@
+---
+applied_at: "2020-10-29"
+applies_to: 
+- api
+- sdks
+is_impactful: true
+is_new_feature: false
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# Changes to OAuth 2 app redirect URL requirements
+
+On October 29th, 2020, Box will begin employing stricter requirements for
+redirect URLs used within new and existing
+[OAuth 2](g://authentication/oauth2/) based Box integrations that may affect
+your application.
+
+Existing application owners that currently use a blank redirect URL in their
+application configuration, as
```

Comment:
How exactly is a blank redirect URI possible? I get a warning when I try to keep it blank.

Reply:
These are apps that were created before the checks were put in place. They were grandfathered in.

Reply:
Ah ok, that makes sense.